crates/starknet/src/starknet/add_declare_transaction.rs

```diff
@@ -12,31 +10,19 @@ pub fn add_declare_transaction_v2(
     starknet: &mut Starknet,
     declare_transaction: DeclareTransactionV2,
 ) -> Result<(TransactionHash, ClassHash)> {
-    let mut declare_transaction = declare_transaction;
-
-    let transaction = DeclareV2::new(
```

Comment:
We don't need this anymore because we are relying on starknet_in_rust's hash calculation?

Reply:
yes